src/System.Collections.NonGeneric/src/System/Collections/Specialized/CollectionsUtil.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace System.Collections.Specialized
 {
-    public class CollectionsUtil
+    public static class CollectionsUtil
```

Comment:
This is *technically* an API breaking change since the class has a public constructor.